SSINS/sky_subtract.py

```diff
@@ -173,6 +193,32 @@ def diff(self):
         super().set_lsts_from_time_array()
         self.data_array = np.ma.masked_array(self.data_array)
 
+    def diff_freq(self):
+
+        """
+        Differences the visibilities in freq. Does so independently for each time.
+        The flags are propagated by taking the boolean OR of the entries that correspond
+        to the visibilities that are differenced from one another. Other metadata
+        attributes are also adjusted so that the resulting SS object passes
+        UVData.check()
+        """
+
+        if self.blt_order != 'baseline':
```

Comment:
I think this reorder isn't necessary. I switch to baseline order so that when I iterate through in `diff`, everything is in a convenient order for the computation. This is because there is no strictly `baseline` or strictly `time` axis in a `UVData` object since not every baseline reports at every time. Rather, they are combined in a `baseline-time` axis that I then reorder. Since the frequency axis is genuinely its own axis, there is no need to reorder.

Reply:
Yeah, this sounds right, I removed it in 176d027d72c1db6b03355102b601be2ddd16ef7a.